Readme.md

```diff
@@ -1,64 +1,114 @@
-# Scalar
-
-[![Build Status](https://dev.azure.com/mseng/Scalar/_apis/build/status/microsoft.scalar?branchName=main)](https://dev.azure.com/mseng/Scalar/_build/latest?definitionId=9297&branchName=main)
+![Scalar](Scalar/Images/scalar-card.png)
 
 ## What is Scalar?
 
-Scalar is a C# application that manages large Git repositories.
-
-Run `scalar register` in an existing Git repo to enable recommended config
-settings and start background maintenance.
-
-If your repo is hosted on a service that supports the
-[GVFS Protocol](https://github.com/microsoft/VFSForGit/blob/HEAD/Protocol.md),
-such as Azure Repos, then `scalar clone <url>` will create a local enlistment with
-abilities like on-demand object retrieval, background maintenance tasks, and
-automatically sets Git config values and hooks that enable performance enhancements.
-Scalar also assists in setting up sparse enlistments.
+Scalar is an opinionated repository management tool. By creating new
+repositories or registering existing repositories with Scalar, your Git
+experience will speed up. Scalar sets advanced Git config settings,
+maintains your repositories in the background, and helps reduce data sent
+across the network.
 
-See [the documentation](docs/index.md) for more details.
+You can learn more about Scalar in this video: [Git at Scale for Everyone](https://www.youtube.com/watch?v=USLB1gwl1vA).
 
 ## Table of Contents
 
-* [Installing on macOS](#installing-on-macos)
-* [Installing on Windows](#installing-on-windows)
-* [Quick start](#quick-start)
+* [Getting Started](#getting-started)
+* [Example Workflow](#example-workflow)
 * [License](#license)
+* [Code of Conduct](#code-of-conduct)
+
+## Getting Started
+
+Full details can be found in [our documentation page](https://github.com/microsoft/scalar/blob/HEAD/docs/index.md).
+
+### Installing on macOS
+
+Scalar is installed and updated on macOS via [Homebrew](https://brew.sh/).
+
+```sh
+brew tap microsoft/git
+brew cask install scalar
+```
+
+If you wish to use the [GVFS Protocol][gvfs-protocol], then you will
+instead need the cask that uses [our custom fork of Git][microsoft-git]:
+
+```sh
+brew tap microsoft/git
+brew cask install scalar-azrepos
+```
+
+When new versions of Scalar are available, you can upgrade in a few
+different ways. First, you can use `brew`:
+
+```sh
+brew update
+brew upgrade --cask scalar[-azrepos]
```

Comment:
Visually, this looks like a typo where the lbracket is stuck to the previous word because we forgot a space.
I'm wondering if this would be better spelled out as 2 separate blocks.